cmd/githubgen/main.go

```diff
@@ -206,3 +258,45 @@ LOOP:
 
 	return nil
 }
+
+func hasMember(id string, checkGithub bool) (bool, error) {
+	if checkGithub {
+		present, err := checkGithubMembership(id)
+		return present, err
+	}
+	for _, m := range members {
+		if id == m {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func inAllowlist(id string) bool {
+	for _, m := range allowlist {
+		if id == m {
+			return true
+		}
+	}
+	return false
+}
+
+func checkGithubMembership(id string) (bool, error) {
+	req, err := http.NewRequest("GET", "https://api.github.com/orgs/open-telemetry/members/"+id, nil)
```

Comment:
This is likely easier to maintain if we use something like https://github.com/google/go-github. Not necessary to do this on this PR, but we should open an issue about it if we don't use it

Reply:
I have filed https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24813 thanks!

Reply:
I've changed the code to use this lib, and we should be good now. Thanks for the suggestion.